sections/tasks.md

```diff
@@ -183,3 +190,62 @@ Destroy task
 **Response:**
 
     Status: 200 OK
+
+Recurring tasks
+------------
+
+Create recurring tasks using `recurring_period` and `anchor_type` in the creation parameters. If a task is recurring, the `recurring-period` will not be nil. 
+
+```
+<task>
+  <body>A task for today</body>
+  <frame>today</frame>
+  <recurring_period>1</recurring_period>
+  <anchor_type>0</anchor_type>
+</task>
+```
+
+The `recurring period` can be one of 6 integers for specific time frames. For example, `0` to recur `Every day`. 
+
+Possible recurring time frames include `Every day`, `Every week`, `Every two weeks`, `Every month`, `Every three months`, `Every six months`, and `Every year`.
+
+
+ ```
+ <select id="task_recurring_period" name="task[recurring_period]"><option value="">No</option>
```

Comment:
i wouldn't show a select menu here since that doesn't really mean anything in this context. i'd turn this into just a bulleted list of values and what they mean